tests/unit/test_adjust_path.py

```diff
@@ -22,12 +33,48 @@ def test_single_node(self):
                 requirements_payload,
                 package_requirement,
                 _file_uploader,
+                api_client,
             )
 
             for job_spec in deployment.get("jobs"):
                 for key, value in job_spec.items():
                     self.assertIsNotNone(value)
 
+    def test_that_upstream_files_are_checked_and_paths_are_properly_re_written(self):
```

Comment:
Added another test to check the files are uploaded and paths are properly re-written